internal/p2p/peermanager.go

```diff
@@ -433,6 +435,7 @@ func (m *PeerManager) Add(address NodeAddress) (bool, error) {
 
 	// else add the new address
 	peer.AddressInfo[address] = &peerAddressInfo{Address: address}
+	fmt.Printf("Adding new peer %s with address %s to peer store\n", peer.ID, address.String())
```

Comment:
nit can we use logger instead of fmt?

Reply:
It seems peermanager doesn't support logger yet, let me check if it's easy to add

Reply:
Added the change to enable logger for peer manager